daml-script/daml/Daml/Script.daml

```diff
@@ -121,6 +124,24 @@ data CommandsF a
 newtype Commands a = Commands (Ap CommandsF a)
   deriving (Functor, Applicative)
 
+data DisclosedContract = DisclosedContract
```

Comment:
I wonder if it would be useful to have `DisclosedContract t` and then an AnyDisclosedContract. Happy to tackle this separately though if at all.

Reply:
Let's explore this in a different PR.